btservice/btservice.cpp

```diff
@@ -33,22 +33,22 @@ int main(int argc, char* argv[])
     btservice::AndroidBluetoothServer androidBluetoothServer;
     if(!androidBluetoothServer.start(address, portNumber))
     {
-        OPENAUTO_LOG(error) << "[btservice] Server start failed.";
+        OPENAUTO_LOG(error) << "[src] Server start failed.";
```

Comment:
Why this change? The old way seems to make more sense.

Reply:
Whoops.  This should not have changed.  Will revert.